-Copy todo-list

Zibilyonik opened the pull request.

Zibilyonik committed:
-Copy todo-list

moise10r reviewed (changes requested):
#  Changes required :recycle:  

## Hey @Zibilyonik ,

![125811485-7f6e14c7-10a8-4701-b6c1-fcb37e47538d](https://user-images.githubusercontent.com/57562869/126295909-fa37d7f1-41e6-48b8-8dd0-0cd8bbf7b284.gif)

You have done a great job so far :clap: :clap: 

 Good points: :+1: 
- Your Readme is professional :heavy_check_mark: 
- Best used of ES6 sytanxes. :heavy_check_mark: 
- You used proper GitHub Flow :heavy_check_mark: 
- Your linter checks are all passing :heavy_check_mark: 


However, there is something that you still need to improve in order to fulfil all requirements!


SUGGESTIONS 
- It's a good practice to commit every time you include the changes in your project.
- Your PR message descriptive should be more descriptive :wink: 

CHANGE REQUIRED
Have a look at the comment below to see what was required

Happy coding computer

Best,
Moise

moise10r commented on src/adjustItems.js:
```diff
@@ -0,0 +1,41 @@
+  }
+  cont.remove();
+  localStorage.clear();
+  localStorage.setItem('todoArray', JSON.stringify(arr));
+}
+
+function editTodo(arr, index) {
+  const obj = document.getElementById(`${index}-text`);
+  arr[index - 1].description = obj.innerText;
+  localStorage.clear();
+  localStorage.setItem('todoArray', JSON.stringify(arr));
+}
+
+function deleteMarked(arr) {
+  const bonk = arr.filter((item) => item.completed === false);
+  arr = [...bonk];
+  localStorage.clear();
+  localStorage.setItem('todoArray', JSON.stringify(arr));
```
It could be better if you could create a function on setting and getting items on the local storage.

Zibilyonik replied:
storageSetter function is now added to address this issue. Thank you for bringing it up 👍

moise10r replied:
That is really nice, I appreciate the effort that you have done so far. Keep it :100: :green_apple:

moise10r reviewed (changes requested):
Hey,
Good job so far! :+1:  fire Your JS code it's excellent and it works! Just there is something to improve in your code to let DRY on the top. :100: 

![125811485-7f6e14c7-10a8-4701-b6c1-fcb37e47538d](https://user-images.githubusercontent.com/57562869/126297367-7d775c6e-faeb-46be-b946-c3308ee0bb56.gif)

Best,
Moise

Zibilyonik committed:
Create storageSetter function for setting localStorage.

Create storageSetter function to replace the old localStorage item with a new one.
